omod/pom.xml

```diff
@@ -17,6 +17,7 @@
         <MODULE_VERSION>${project.parent.version}</MODULE_VERSION>
         <MODULE_PACKAGE>${project.parent.groupId}.${project.parent.artifactId}</MODULE_PACKAGE>
         <WEBSERVICES_VERSION>2.13</WEBSERVICES_VERSION>
```

Comment:
We do not encourage referencing difference versions of the same module like you have done here by adding another version.

Reply:
Deleted the 2.20.0 version. Only the 2.13 version remains